File: php-slim/lib/Model/ResponseRoute.php
<?php
/**
 * ResponseRoute
 */
namespace OpenAPIServer\Model;

/**
 * ResponseRoute
 */
class ResponseRoute
{
    
    /** @var \DateTime $departureTime */
    private $departureTime;
    
    /** @var \DateTime $arrivalTime */
    private $arrivalTime;
    
    /** @var \OpenAPIServer\Model\ResponseRoutePart[] $parts */
    private $parts;
}

File: fsharp-giraffe-server/OpenAPI/src/model/ResponseRoute.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open OpenAPI.Model.ResponseRoutePart

module ResponseRoute = 

  //#region ResponseRoute


  type ResponseRoute = {
    DepartureTime : DateTime;
    ArrivalTime : DateTime;
    Parts : ResponseRoutePart[];
  }
  //#endregion
  